### PR TITLE
Release notes for 0.4.0

### PR DESCRIPTION
**Pull request by Gallaecio**

### CODE REVIEW

**Comment by wRAR** [comment]
Looks like the CI config is incorrect

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,23 @@
 Changes
 =======
 
+0.4.0 (2024-06-DD)
+------------------
+
+* Dropped official support for Python 3.6 and lower, and added official support
```
Comment (wRAR) [inline CHANGES.rst]: I think 3.7 was also dropped

Comment (Gallaecio) [reply CHANGES.rst]: Well, technically it was never officially supported :smile:

**Review by wRAR — APPROVED** [review]

File: CHANGES.rst (hunk 1 above)
Comment (kmike) [reply CHANGES.rst]: Nice one :) 
I think it's still better to say this explicitly - e.g. mention new min and max Python versions.

**Pull request merged**